source/lovelace/index.markdown

```diff
@@ -111,3 +111,14 @@ It is possible to add your own custom cards to show up in the Lovelace UI. For m
 ## Current limitations
 
 This is the very very early version aimed at gathering feedback. Discussion and suggestions are welcome in the [ui-schema repository](https://github.com/home-assistant/ui-schema) and in the [chat](/join-chat/) in #lovelace.
+
+## FAQ
```

Comment:
Make it a linkable title

Reply:
It should be using https://www.home-assistant.io/lovelace/#faq considering https://www.home-assistant.io/lovelace/#custom-cards is also available. Is there any special function the "linkable" does?

Reply:
linkable title will create the #link, also its in the docs standard: https://developers.home-assistant.io/docs/en/documentation_standards.html

The other titles in this file already use linkable title.